### PR TITLE
Get tenant alias from penpal

### PR DESCRIPTION
**Pull request by ChristopherRDunn**
Uses penpal to get the tenantAlias and use it to store the config data in the db, [Get saasquatch tenant alias from penpal](https://trello.com/c/2VCl8D2B) 
Also fixes an issue where we had an infinitely expanding iframe, [Figure out the iframe height bug](https://trello.com/c/IfHlF6JZ)

The best way to test this is in the iframe on SaaSquatch's website.
_If the integration has a scrollbar, then it's running a different branch. Message me and I can redeploy it on this branch._

The biggest issue with penpal is it doesn't let our application run outside of the SaaSquatch integration's context. It makes running the frontend locally very difficult. I updated our ReadMe to include details on how to do it

### CODE REVIEW

**Review by nicolepeverley — APPROVED** [review]
Looks good to me thanks for all your work on this 🚀

**Review by lucianojd — APPROVED** [review]
Looks good! Great job Chris!

File: frontend/src/components/ConfigurationP1.tsx [hunk 1]
```diff
@@ -135,7 +145,9 @@ export function Controller(){
   // Gets config data on page load
   useEffect(() => {
     const getConfigData = () => {
-      axios.get(API_CONFIGURATION_URL)
+      axios.get(API_CONFIGURATION_URL,
+        { params: {token: document.cookie, SaaSquatchTenantAlias: config.saasquatchTenantAlias} }
```
Comment (nicolepeverley) [inline frontend/src/components/ConfigurationP1.tsx]: I realized you can safely remove `token: document.cookie` now I changed the way this works with the fix!

File: frontend/src/components/ConfigurationP2.tsx [hunk 2]
```diff
@@ -146,9 +155,11 @@ export function Controller(){
   // Gets config data on page load
   useEffect(() => {
     const getConfigData = () => {
-      axios.get(API_CONFIGURATION_URL)
+      axios.get(API_CONFIGURATION_URL,
+        { params: {token: document.cookie, SaaSquatchTenantAlias: config.saasquatchTenantAlias} }
```
Comment (nicolepeverley) [inline frontend/src/components/ConfigurationP2.tsx]: Here too!

**Pull request merged**